Voice Controlled Car

praveenkumar1410 opened the pull request:
## Describe your project in 1-3 sentences

<!-- I am building a LED light display that can be controlled through an app. -->

## Hack Club Slack handle

<!-- Join the Hack Club Slack at https://hackclub.com/slack -->

## Checklist before submitting proposal:

- [x] I am a current high school student (or younger).
- [x] I have submitted an RSVP at <https://hackclub.com/winter>
- [x] I am a member of the Hack Club Slack (Join at <https://hackclub.com/slack>)
- [x] I have self-reviewed my project proposal.

bellesea reviewed (changes requested):
hiii @praveenkumar1410 omg this is aewsome!! 

are you referencing any guides / existing voice controlled cars for this?

and, what's the role of the pi4 in this project?

also, what do you mean by "generate a g-code for the image and then run the machine"? and, is there an existing tool that allows you to parse the voice commands into instructions for the car or will you be coding that yourself?

thank you so much!!

praveenkumar1410 commented:
> hiii @praveenkumar1410 omg this is aewsome!!
> 
> are you referencing any guides / existing voice controlled cars for this?
> 
> and, what's the role of the pi4 in this project?
> 
> also, what do you mean by "generate a g-code for the image and then run the machine"? and, is there an existing tool that allows you to parse the voice commands into instructions for the car or will you be coding that yourself?
> 
> thank you so much!!

Hii ,I'm @praveenkumar1410 

Yes , I am refering an existing product. The project is available in arduino.

The pi4 helps to connect with wifi , so that it will be helpful for me to control it using mobile application( which as joy stick options)/ I can also control it using internet. 

The same car can be used for many innovative ideas with pi4.

"generate a g-code for the image and then run the machine" - I recently heard about it generating code when we specify actions. But I haven't used it . I am going to code on my own.

Thank you😊

praveenkumar1410 reviewed (commented):
updated praveenkumar1410.md

bellesea commented:
@praveenkumar1410 great! thanks for adding this, could you please link the existing project that you're modelling your car after within your file?

and, could you please update your plan to show when you'd use the Pi, how it integrates with your car, and steps you'll need to do to allow you to control it from your mobile app?

also, how will your dashboard generate g-code? 

and, i haven't looked that much into how voice controlled cars work... would you be willing to expand on how your car would follow g-code instructions? 

please let me know if you have any questions!

praveenkumar1410 commented:
Hii @bellasea Thank you for notifying. 

Sorry for inconvenience.

I have changed my document in response to your request.

There is no generation of G-code. I am coding on my own. I removed that part from my document. 

Please check it out.

bellesea commented on praveenkumar1410.md:
```diff
@@ -0,0 +1,58 @@
+
+## Existing Project
+
+https://create.arduino.cc/projecthub/mathwuy/voice-controlled-k-nex-car-f1b867?ref=similar&ref_id=443067&offset=4
```
for some reason whenever i go to this link it redirects to https://projecthub.arduino.cc/ :( would you be able to share a downloaded pdf or smth of the page so we could view it as reference?

bellesea replied:
is it the same as https://www.hackster.io/mathwuy/voice-controlled-k-nex-car-f1b867?

bellesea commented on praveenkumar1410.md:
```diff
@@ -0,0 +1,58 @@
+| Raspberry Pi 4                        | https://bit.ly/3GaOxZY                                                                         | $47.15 |
+| Total                                 |                                                                                                | $200.98|
+
+No shipping costs are included because the prices included in the table is very similar to the prices in my local electronic store. I can confirm every item listed here is available in the store and don't need to ship them from overseas.
```
does the budget include taxes?

bellesea commented on praveenkumar1410.md:
```diff
@@ -0,0 +1,58 @@
+
+2.) flash the 16 gb sd card with the image.
+
+3.) Plug the monitor (via HDMI), mouse, keyboard, Arduino Uno (via USB), and WiFi adapter (if applicable) into the Raspberry Pi.
```
do you already have all of these?

bellesea reviewed (changes requested):
@praveenkumar14 apologies for the delayed response! thanks sm for pushing these new changes :))) just a few more questions hehe :)

thanks!

cfanoulis commented:
Hi @praveenkumar1410! Thanks for your proposal. It's been a while since we've had any news from you and we want to know if you're still interested in the grant. Please reply in the next 48 hours to this PR (or this email) if you're still interested, else we'll be closing this proposal. Hacky New Year!

cfanoulis commented:
Hello! Unfortunately 2 days have passed, which means that I'll be closing this PR as stale. If you still wish to pursue a grant please let us know by replying to this PR before Jan. 15th
